fp.sql

```diff
@@ -70,17 +82,86 @@ gather_contact_data AS (
     GROUP BY
         site_short
 ),
-join_data AS (
+--For KPI % of students admitted to Best-Fit College choose to enroll at Best-Fit College
+gather_best_fit_data AS (
+    SELECT
+        contact_id,
+        site_short,
+        --Students accepted to Best Fit
+        (
+            SELECT
+                student_c
+            FROM
+                `data-warehouse-289815.salesforce_clean.college_application_clean` AS subq1
+            WHERE
+                (
+                    admission_status_c = "Accepted"
```

Comment:
@isabelreyna this seems different than the options we've used before to signify acceptances. Just flagging in case
<img width="302" alt="Screen Shot 2021-06-08 at 1 53 55 PM" src="https://user-images.githubusercontent.com/38270713/121256121-14b94080-c861-11eb-8845-093dfbd67afb.png">